test_workloads.sh

```diff
@@ -70,10 +91,11 @@ run_test() {
     
     sleep 3
   elif [ $db == redis ]; then
+    echo "Stopping redis server"
     # Stop redis server
     kill $(pgrep redis-server)
 
-    sleep 3
+    sleep 15
```

Comment:
That's huge. Is it really needed based on your experiments? What error are you getting with lower sleeping times?